src/Sulu/Bundle/ContactBundle/Controller/AccountMediaController.php

```diff
@@ -49,6 +49,20 @@ public function postAction($id, Request $request)
     }
 
     /**
+     * Replaces the medias of the account.
+     *
+     * @param $id
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function putAction($id, Request $request)
```

Comment:
this leads to the api interface:
GET| /api/accounts/1/medias | content: mediaIds | returns the medias which are assigned to account 1 as list
PUT | /api/accounts/1/medias | content: mediaIds | sets the medias of the account 1 to the mediaIds of the request body
POST | /api/accounts/1/medias | content: mediaId | adds the media with the id of the request body to the account 1
DELETE| /api/accounts/1/medias/2 | content: empty | removes the media with the id 2 from the account 1